config/config.example.yml

```diff
@@ -169,6 +169,16 @@ browseBy:
   # The absolute lowest year to display in the dropdown (only used when no lowest date can be found for all items)
   defaultLowerLimit: 1900
 
+communityList:
+  # No. of communities to list per expansion (show more)
+  pageSize: 20
+
+home:
```

Comment:
@mark-cooper : Quick comment...Could we change this to say `homePage:`?  We have a different PR (#1801) which is creating a section called `homePage` and it'd be good to align these two PRs so that we have a single section for all homepage configs.

It's unclear which PR will be ready to merge first, but both PRs also have a `homepage-config.interface.ts` sort of file...and we'll need to find a way to merge the two together.

Reply:
@tdonohue no problem! Also I updated things to be more aligned with #1801 so there's less to do bringing them together.